### DISCUSSION TITLE
Where and How to Add Symlinks OM build.gradle?

### DISCUSSION THREAD

**Discussion by LaachirDeeper** [organicmaps/organicmaps#6580] [Q&A]
Hello Devs, can you help please to solve this selected line build.gradle Organic maps, I try importing the OM into android studio so that I can contribute. Thanks.

**The number 20 to 24 line:**
// Please add symlinks to google/java/app/organicmaps/location for each new gms-enabled flavor below:
  // ```
  // mkdir -p src/$flavor/java/app/organicmaps/
  // ln -sf ../../../../google/java/app/organicmaps/location src/$flavor/java/app/organicmaps/
  // ls -la src/$flavor/java/app/organicmaps/location/GoogleFusedLocationProvider.java

![2023-11-16_173008.jpg](https://github.com/organicmaps/organicmaps/assets/25303041/62c2d539-ada0-4759-a48e-69bb3008a52d)

**Comment by biodranik** [comment]
These lines are commented. What is your question about?

**Reply by LaachirDeeper** [reply]
Alright :) I just saw like they commented Please Add symlinks, I thought it's a command for in order to import the code successful, Also it mentioned in [INSTALL.md](https://github.com/organicmaps/organicmaps/blob/master/docs/INSTALL.md) page that's all 👍 But I fix the problem.
